core/base/module.php

```diff
@@ -280,6 +280,18 @@ final protected function get_css_assets_url( $file_name, $relative_url = null, $
 		return $this->get_assets_url( $file_name, 'css', $relative_url, $add_min_suffix );
 	}
 
+	final protected function enqueue_module_assets( $dependencies = [] ) {
```

Comment:
Move to nested infra

Reply:
https://github.com/elementor/elementor/pull/18032